Update user-settings.mdx (#8011)
newrelic/docs-website: src/content/docs/alerts-applied-intelligence/applied-intelligence/incident-intelligence/user-settings.mdx

An issue is considered inactive when one of these things occurs:

* The issue has collected 5,000 incidents.

> An issue can be inactive for up to a maximum of 30 days, before it's automatically closed.

Reviewer: ```suggestion
After 30 days, an inactive issue will be automatically closed.
```
